test/integration/testpaths/testpath_disable_enable_zone.py

```diff
@@ -0,0 +1,485 @@
+            diskofferingid=self.disk_offering.id
+        )
+        self.assertNotEqual(
+            data_volume,
+            None,
+            "Check if volume gets created"
+        )
+
+        ISO = Iso.create(
+            self.apiclient,
+            self.testdata["iso2"],
+            zoneid=self.zone.id,
+            account=self.account.name,
+            domainid=self.account.domainid,
+        )
+
+        self.assertNotEqual(
+            ISO,
+            None,
+            "Check if volume gets created"
+        )
+        root_vm_new.delete(self.apiclient)
+        # Non root user
+        user_vm_new = VirtualMachine.create(
+            self.userapiclient,
+            self.testdata["small"],
+            templateid=self.template.id,
+            accountid=self.account.name,
+            domainid=self.account.domainid,
+            serviceofferingid=self.service_offering.id,
+            zoneid=self.zone.id,
+            mode=self.zone.networktype)
+
+        self.assertNotEqual(user_vm_new,
+                            None,
+                            "Verify that admin should create new VM")
+
+        snap = Snapshot.create(
+            self.userapiclient,
+            root_volume[0].id)
+
+        self.assertNotEqual(snap,
+                            None,
+                            "Verify that admin should snashot")
+
+        snapshots = list_snapshots(
+            self.userapiclient,
+            volumeid=root_volume[0].id,
+            listall=True)
+
+        template_regis = Template.register(
+            self.userapiclient,
+            self.testdata["template_2"],
+            zoneid=self.zone.id)
+
+        self.assertNotEqual(
+            template_regis,
+            None,
+            "Check if template gets created"
+        )
+        self.assertNotEqual(
+            template_from_snapshot,
+            None,
+            "Check if template gets created"
+        )
+
+        data_volume = Volume.create(
+            self.userapiclient,
+            self.testdata["volume"],
+            zoneid=self.zone.id,
+            account=self.account.name,
+            domainid=self.account.domainid,
+            diskofferingid=self.disk_offering.id
+        )
+        self.assertNotEqual(
+            data_volume,
+            None,
+            "Check if volume gets created"
+        )
+
+        ISO = Iso.create(
+            self.userapiclient,
+            self.testdata["iso2"],
+            zoneid=self.zone.id,
+            account=self.account.name,
+            domainid=self.account.domainid,
+        )
+
+        self.assertNotEqual(
+            ISO,
+            None,
+            "Check if volume gets created"
+        )
+        user_vm_new.delete(self.apiclient)
+        # Step 3
+        # Deletion of zone should fail if vm,volume is present on the zone
+        with self.assertRaises(Exception):
+            self.zone.delete(self.apiclient)
+        return
```

Comment:
Move this test path to /test/integration/component/maint/ as it might affect other test cases when run in parallel.